lib/l10n/app_de.arb

```diff
@@ -108,11 +108,35 @@
 
   "pageHelpTitle": "Hilfe",
   "pageHelpContactUs": "Kontaktiere uns",
-  "pageHelpWhatIsMotisQuestion": "Was kann ich mit der MOTIS Ride App machen?",
-  "pageHelpWhatIsMotisAnswer": "Mit der MOTIS Ride App kannst du einige nützliche Dinge machen.",
-  "pageHelpWhereIsAccessibilityQuestion": "Wo finde ich die Einstellungen für Barrierefreiheit?",
-  "pageHelpWhereIsAccessibilityAnswer": "Diese App unterstützt eine Vielzahl von Einstellungen zur Barrierefreiheit, die vom Betriebssystem angeboten werden. Geh zu den Einstellungen deines Betriebssystems, um Screen Reader oder größere Schrift zu aktivieren!",
-
+  "pageHelpEmailSubject": "Anfrage von User-ID [{id}]",
+  "pageHelpQuestion1": "Was ist MOTIS?",
+  "pageHelpAnswer1": "MOTIS umfasst viele Tools zur Unterstützung des intermodalen Routings. Das Ziel ist es, bei der Planung von Reisen zu helfen, die öffentliche Verkehrsmittel, Bike-Sharing, Ride-Sharing und weitere Transportmittel umfassen.",
+  "pageHelpQuestion2": "Wie viel kosten Fahrten? Wie kann ich bezahlen?",
+  "pageHelpAnswer2": "Wir haben den Zahlungsprozess so konzipiert, dass er sich vollständig auf Benutzervereinbarungen stützt. Sowohl die Kosten als auch die Zahlungsmethode einer Fahrt können persönlich oder in unserem In-App-Chat ausgehandelt werden. Als Ausgangspunkt sehen Sie für die meisten Fahrten einen Preisvorschlag.",
+  "pageHelpQuestion3": "Wir haben uns auf einen Preis geeinigt, aber jetzt zahlt mein Fahrer ihn nicht. Was kann ich machen?",
+  "pageHelpAnswer3": "Wir entschuldigen uns für die Unannehmlichkeiten. Leider können wir Ihnen in diesem Fall nur die Möglichkeit bieten, Benutzer zu melden. Tippen Sie in seinem Profil auf die Schaltfläche \"Meldung\" und stellen Sie sicher, dass Sie den Grund \"Hat nicht bezahlt\" auswählen.",
```

Comment:
these strings might be dangerous when we choose to rename the button text or the reason. any way around that?

Reply:
Don't think so. You would have to reference another string from the arb file and I haven't seen any documentation for that

Reply:
Well you could load the string in dart and then give it as a parameter